core-backend/common/src/funcs.rs

```diff
@@ -16,44 +16,28 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-use crate::{GAS_ALLOWANCE_STR, LEAVE_TRAP_STR, WAIT_TRAP_STR};
 use alloc::{vec, vec::Vec};
 use gear_core::memory::Memory;
+use gear_core_errors::MemoryError;
```

Comment:
```suggestion
use gear_core::memory::{Memory, Error as MemoryError};
```

Reply:
Is this so important?

Reply:
Of course, in comparison to significant changes that you did, this is minor stuff. During review we see minor/major things to be changed. This is minor stuff and you can leave it how it is. Or you can emphasize using re-export. :)

Reply:
> During review I see minor/major things to be improved

Don't be afraid to say about them. I just wondering sometimes